collectors/Makefile.am

```diff
@@ -3,30 +3,41 @@
 MAINTAINERCLEANFILES = $(srcdir)/Makefile.in
 
 SUBDIRS = \
-    plugins.d \
-    apps.plugin \
-    cgroups.plugin \
-    charts.d.plugin \
-    checks.plugin \
-    cups.plugin \
-    diskspace.plugin \
-    fping.plugin \
-    ioping.plugin \
-    freebsd.plugin \
-    freeipmi.plugin \
-    idlejitter.plugin \
-    macos.plugin \
-    nfacct.plugin \
-    xenstat.plugin \
-    perf.plugin \
-    node.d.plugin \
-    proc.plugin \
-    python.d.plugin \
-    slabinfo.plugin \
-    statsd.plugin \
-    tc.plugin \
-   	$(NULL)
+	plugins.d \
+	apps.plugin \
+	cgroups.plugin \
+	charts.d.plugin \
+	checks.plugin \
+	cups.plugin \
+	diskspace.plugin \
+	fping.plugin \
+	ioping.plugin \
+	freebsd.plugin \
+	freeipmi.plugin \
+	idlejitter.plugin \
+	macos.plugin \
+	nfacct.plugin \
+	xenstat.plugin \
+	perf.plugin \
+	node.d.plugin \
+	proc.plugin \
+	python.d.plugin \
+	slabinfo.plugin \
+	statsd.plugin \
+	tc.plugin \
+	$(NULL)
+
```

Comment:
It looks as if you have changed the indent from 4 spaces to 6 spaces here. Is this intentional?